content/manuals/subscription/change.md

```diff
@@ -0,0 +1,114 @@
+4. Select the account you want to upgrade in the drop-down at the top-left of the page.
+5. Select **Upgrade**.
+6. Follow the on-screen instructions to complete your upgrade.
+
+{{< /tab >}}
+{{< tab name="Legacy Docker plan" >}}
+
+You can upgrade a legacy Docker Core, Docker Build Cloud, or Docker Scout subscription plan to a Docker subscription plan that includes access to all tools.
+
+Contact [Docker sales](https://www.docker.com/pricing/contact-sales/) to upgrade your legacy Docker plan.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+## Downgrade your subscription
+
+You can downgrade your Docker subscription at anytime before the renewal date. The unused portion of the subscription isn't refundable or creditable.
+
+When you downgrade your subscription, access to paid features is available until the next billing cycle. The downgrade takes effect on the next billing cycle.
+
+> [!IMPORTANT]
+>
+> If you downgrade your personal account from a Pro subscription to a Personal subscription, note that [Personal subscriptions](details.md#docker-personal) don't include collaborators for private repositories. Only one private repository is included with a Personal subscription. When you downgrade, all collaborators will be removed and additional private repositories are locked.
+> Before you downgrade, consider the following:
+> - Team size: You may need to reduce the number of team members and convert any private repositories to public repositories or delete them. For information on features available in each tier, see [Docker Pricing](https://www.docker.com/pricing).
+> - SSO and SCIM: If you want to downgrade a Docker Business subscription and your organization uses single sign-on (SSO) for user authentication, you need to remove your SSO connection and verified domains before downgrading. After removing the SSO connection, any organization members that were auto-provisioned (for example, with SCIM) need to set up a password to sign in without SSO. To do this, users can [reset their password at sign in](/accounts/create-account/#reset-your-password-at-sign-in).
+
+{{< tabs >}}
+{{< tab name="Docker plan" >}}
+
+If you have a [sales-assisted Docker Business subscription](details.md#sales-assisted), contact your account manager to downgrade your subscription.
+
+To downgrade your Docker subscription:
+
+1. Sign in to [Docker Home](https://app.docker.com/).
+2. Under Settings and administration, select **Billing**.
+3. Select the account you want to downgrade in the drop-down at the top-left of the page.
+4. Select the action icon and then **Cancel subscription**.
+5. Review the cancellation warnings, then select **Confirm cancellation**.
+6. Optional. Fill out the feedback survey, or select **Skip**.
+
+{{< /tab >}}
+{{< tab name="Legacy Docker plan" >}}
+
+If you have a [sales-assisted Docker Business subscription](details.md#sales-assisted), contact your account manager to downgrade your subscription.
+
+### Downgrade Docker Core subscription
```

Comment:
Title still a mentions the term "Docker Core" instead of Legacy Docker, intentional?